src/pages/blog/technical-121s.md

```diff
@@ -0,0 +1,95 @@
+---
+author: 'jms'
+layout: '../../layouts/BlogPost.astro'
+title: 'Ways to improve one-to-ones for technical people-managers'
+description: "An experience report of actionable ideas for you and your mentees to get more from your regular one-to-ones."
+category: 'culture'
+publishedDate: '2024-03-01'
+tags:
+  - management skills
+  - culture
+  - development teams
+  - mentoring
+heroImage: 'technical-121s.jpg'
+---
+
+I've just come out of a JUXT team-leads catch-up where we've discussed the art of 1-to-1s, mostly to share experiences and best practices from our wide variety of clients and projects, but with the aim of eventually writing up an internal beginners' guide for new team-leads.
+
+Anyway, it got me thinking, reflecting on what I do and why.
+I highly doubt anything in here is particularly novel or ground-breaking, but I hope they're as useful to you as they have been to me!
```

Comment:
`they're` -> `these reflections` perhaps?

Reply:
@refset thanks, fair call to clarify - intended "they" as in "the ideas", if that makes any difference?

Reply:
But "the ideas" aren't mentioned at the moment @jarohen. It's a floating pronoun.

Reply:
@joelittlejohn fair, yep :slightly_smiling_face: will be more explicit